server/@types/express/index.d.ts

```diff
@@ -28,14 +27,12 @@ declare module 'express-session' {
   interface SessionData {
     returnTo: string
     nowInMinutes: number
+    prisonerListSortOptions: string
```

Comment:
Moved this (`prisonerListSortOptions`) up in the order, away from the other objects we'll look at in due course.
`prisonerListSortOptions` is NOT prisoner centric and is used to retain the sort options on the prisoner list screen (so that when you come back to prisoner list screen after working on a given prisoner you keep the original sort order etc)

(It's the kind of thing that perhaps could/should have been implemented with a cookie, rather than holding it in the session, but I think cookies are perhaps frowned upon here and we would have to declare it in a cookie usage type page 🤷‍♂️ )

Reply:
Yes something to think about. Cookies are used elsewhere in moj so perhaps it's not the worst idea. Certainly if it was a modern webapp it'd be something stored client side unless it was going to persist across multiple user-sessions.